subscription-manager.spec

```diff
@@ -296,6 +296,7 @@ rm -rf %{buildroot}
 # yum plugin config
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/subscription-manager.conf
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/product-id.conf
+%config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/search-disabled-repos.conf
```

Comment:
It does show a warning: "Plugin "search-disabled-repos" requires API 2.7. Supported API is 2.6.".

If you really don't want that warning you can require a specific yum version, but yum versions providing this API version will be different on RHEL7 and RHEL6.

If you disable the plugin by default, how would you handle enabling it when the correct yum version gets installed?